recipes/hass

```diff
@@ -0,0 +1 @@
+(hass :version-regexp "v\\(.*\\)" :fetcher github :repo "purplg/hass")
```

Comment:
```suggestion
(hass :fetcher github :repo "purplg/hass")
```
I believe you wouldn't need to specify `:version-regexp` in your recipe -- the one you're using is compatible with MELPA's defaults